core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -486,9 +486,10 @@ object ConsumerGroupCommand extends Logging {
   }
 
   class ConsumerGroupCommandOptions(args: Array[String]) {
-    val ZkConnectDoc = "REQUIRED (only when using old consumer): The connection string for the zookeeper connection in the form host:port. " +
-      "Multiple URLS can be given to allow fail-over."
-    val BootstrapServerDoc = "REQUIRED (unless old consumer is used): The server to connect to."
+    val ZkConnectDoc = "REQUIRED (unless the 'bootstrap-server' option is specified): The connection string for the zookeeper connection in the form host:port. " +
```

Comment:
It seems important to emphasize that this option is for the old consumer. Could we simply say "Required for consumer groups based on the old consumer" and leave out the mention of the `bootstrap-server` option? Similar for the bootstrap server option.